anomaly-detection/src/main/java/io/pravega/anomalydetection/event/pipeline/PravegaEventPublisher.java

```diff
@@ -0,0 +1,76 @@
+/**
+ *
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries.
+ *
+ */
+package io.pravega.anomalydetection.event.pipeline;
+
+import io.pravega.anomalydetection.event.AppConfiguration;
+import io.pravega.anomalydetection.event.producer.ControlledSourceContextProducer;
+import io.pravega.anomalydetection.event.producer.SourceContextProducer;
+import io.pravega.anomalydetection.event.serialization.PravegaSerializationSchema;
+import io.pravega.anomalydetection.event.state.Event;
+import io.pravega.client.stream.impl.JavaSerializer;
+import io.pravega.connectors.flink.FlinkPravegaWriter;
+import io.pravega.connectors.flink.PravegaEventRouter;
+import io.pravega.connectors.flink.PravegaWriterMode;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URI;
+
+public class PravegaEventPublisher implements IPipeline {
+
+	private static final Logger LOG = LoggerFactory.getLogger(PravegaEventPublisher.class);
+
+	@Override
+	public void run(AppConfiguration appConfiguration) throws Exception {
+		publishUsingFlinkConnector(appConfiguration);
+	}
+
+	private void publishUsingFlinkConnector(AppConfiguration appConfiguration) throws Exception {
```

Comment:
It is option. We can either run this as a Flink job (in that case error records will be generated at random cycles) or standalone (where we can simulate error record generation on demand)